collect_app/build.gradle

```diff
@@ -278,6 +278,7 @@ dependencies {
     implementation("org.getodk:javarosa:3.2.0-SNAPSHOT") {
         exclude group: 'joda-time'
         exclude group: 'org.slf4j'
+        exclude group: 'org.hamcrest', module: 'hamcrest-all'
```

Comment:
Not sure why JavaRosa exposes this. We definitely need to look at why we end up with so many excludes here.